src/utils/date.ts

```diff
@@ -78,6 +79,10 @@ export const getRelativeDate = (daysAgo: number) => {
   return formattedDate(date);
 };
 
+export const getRelativeTimeStamp = (daysAgo: number) => {
+  return Date.now() - daysAgo * 24 * 60 * 60 * 1000;
+};
```

Comment:
There is a js lib called `date-fns` to perform these kind of operations on dates. Check that out

Reply:
I looked in to it but this is mainly for formatting the date to the wanted format. but I want the number timestamp here.